include/openPMD/regions/Box.hpp

```diff
@@ -0,0 +1,437 @@
+#ifndef REGIONS_BOX_HPP
+#define REGIONS_BOX_HPP
```

Comment:
We usually use `#pragma once` for these.

Reply:
Done.